test/graphql-assertions.test.js

```diff
@@ -73,7 +73,7 @@ test('assertError throws if no error matches the path', async (test) => {
   test.throws(() => assertError(response, 'some.other.path', 'boom!'), `No error found with path 'some.other.path'. The paths with errors were: error`);
 });
 
-test('assertError throws if the error does not match the message', async (test) => {
+test.skip('assertError throws if the error does not match the message', async (test) => {
```

Comment:
I'm not sure what your plan was, but I don't think we'd merge with a skip in place.